src/test/java/pageObjects/chpldownload_page.java

```diff
@@ -0,0 +1,29 @@
+package pageObjects;
+import static org.testng.Assert.assertTrue;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import stepDefinitions.hooks;
+
+public class chpldownload_page {
+	
+	private WebDriver driver;
+	private static WebElement element = null;
+	
+	public static void print_page_title(WebDriver driver) {
```

Comment:
This looks like the kind of thing that should be in some "helper" class. It's not specific to the Download page, but could be used on any page and just return the title as a String for tests to use to verify stuff